operators/pkg/apis/common/v1alpha1/common.go

```diff
@@ -59,11 +61,14 @@ type SubjectAlternativeName struct {
 
 // HTTPService contains defaults for a HTTP service.
 type HTTPService struct {
```

Comment:
Can we remove this struct entirely and directly rely on `corev1.Service`?

Reply:
It would contain `.Status` as well, which seems a little odd? Going through a `HTTPService` thing makes sense, in the same vein as `PodTemplate` does.

Reply:
Renaming to `ServiceTemplate`

Reply:
👍